tidb-cloud/connect-to-tidb-cluster-serverless.md

```diff
@@ -5,29 +5,57 @@ summary: Learn how to connect to your TiDB Serverless cluster via different meth
 
 # Connect to Your TiDB Serverless Cluster
 
-This document introduces the methods to connect to your TiDB Serverless cluster.
+This document describes how to connect to your TiDB Serverless cluster.
 
 > **Tip:**
 >
 > To learn how to connect to a TiDB Dedicated cluster, see [Connect to Your TiDB Dedicated Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
 
-After your TiDB Serverless cluster is created on TiDB Cloud, you can connect to it via one of the following methods:
+## Connection methods
 
-- [Connect via private endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) (recommended)
+This section describes the connection methods you can use to connect to a TiDB Serverless cluster.
 
-    Private endpoint connection provides a private endpoint to allow SQL clients in your VPC to securely access services over AWS PrivateLink, which provides highly secure and one-way access to database services with simplified network management.
+- Direct connections
 
-- [Connect via public endpoint](/tidb-cloud/connect-via-standard-connection-serverless.md)
+  Direct connections mean the MySQL native connection system over TCP, You can connect to your TiDB Serverless cluster using any tool that supports MySQL connection, such as [MySQL client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html).
 
-    The standard connection exposes a public endpoint with traffic filters, so you can connect to your TiDB cluster via a SQL client from your laptop.
+- [Data Service (beta)](/tidb-cloud/data-service-overview.md)
 
-    TiDB Serverless supports [TLS connections](/tidb-cloud/secure-connections-to-serverless-clusters.md), which ensures the security of data transmission from your applications to TiDB clusters.
+  TiDB Serverless provides a Data Service feature that enables you to connect to your TiDB Serverless cluster via an HTTPS request using a custom API endpoint. Different from direct connections, Data Service access TiDB Serverless data via restful api rather than the raw SQL.
```

Comment:
```suggestion
  TiDB Serverless provides a Data Service feature that enables you to connect to your TiDB Serverless cluster via an HTTPS request using a custom API endpoint. Unlike direct connections, Data Service accesses TiDB Serverless data via restful API rather than the raw SQL.
```